addon/mixins/osf-model.js

```diff
@@ -7,5 +7,37 @@ import DS from 'ember-data';
 
 export default Ember.Mixin.create({
     links: DS.attr('links'),
-    embeds: DS.attr('embed')
+    embeds: DS.attr('embed'),
+
+    relationshipLinks: DS.attr('links'),
+    dirtyRelationships: Ember.computed('_dirtyRelationships', function() {
+        var dirtyRelationships = this.get('_dirtyRelationships');
+        return Object.keys(dirtyRelationships).map((rel) => {
+            if (rel === 'files' || rel === 'node' || rel === 'user') {
```

Comment:
That was because our dirty-ing method was catching those fields as well, and they are not fields that would be updated via relationships anyways (we have children and contributor for that)